### PR TITLE
CHE-896 cleaning custom colors in SVN plugin

### PR DESCRIPTION
**Pull request by stour**
Signed-off-by: Stéphane Tournié stephane.tournie@serli.com

@skabashnyuk @vparfonov

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ok. @vparfonov ?

File: core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java [hunk 1]
```diff
@@ -1465,4 +1465,12 @@
     String gitConsoleStagedFilesColor();
     String gitConsoleUnstagedFilesColor();
     String gitConsoleErrorColor();
+
```
Comment (vparfonov) [inline core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java]: Looks here problem with styles, not good add SVN colors to the IDE API need find other solution. The same problem with GIT. @ashumilova what you think about it?

Comment (stour) [inline core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java]: What about having styles specific to VCS and not to Git and Subversion ?
So in this case, replace above and below lines with:

``` java
String vcsConsoleStagedFilesColor();
String vcsConsoleUnstagedFilesColor();
String vcsConsoleErrorColor();
String vcsConsoleModifiedFilesColor();
String vcsConsoleChangesLineNumbersColor();
```

Comment (vparfonov) [inline core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java]: better for me

Comment (stour) [inline core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java]: ok I will do this change

**Comment by stour** [comment]
@vparfonov PR updated

**Comment by vparfonov** [comment]
ok

**Pull request merged**